### PR TITLE
Ra 1034

### PR DESCRIPTION
**Pull request by VoldarGames**

### CODE REVIEW

File: VAS.Core/App.cs [hunk 1]
```diff
@@ -43,6 +43,8 @@ public abstract class App
 		public IHotkeysService HotkeysService;
 		public IResourcesLocator ResourcesLocator;
 		public IFileSystemManager FileSystemManager;
+		public string Website;
```
Comment (vmartos) [inline VAS.Core/App.cs]: where is Website Set?

File: VAS.Services/ViewModel/AboutVM.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	/// About ViewModel, is needed because of AboutState requires a ViewModel
+	/// </summary>
+	public class AboutVM : ViewModelBase
+	{
+		Version version = App.Current.Version;
+
+		public AboutVM ()
+		{
+			ProgramName = App.Current.SoftwareName;
+			Version = $"{version.Major}.{version.Minor}.{version.Build}";
```
[the remark this answers is not in the document]
Comment (VoldarGames) [reply VAS.Services/ViewModel/AboutVM.cs]: I think string interpolation is much clearer here than ToString(3)

File: VAS.Core/App.cs (hunk 1 above)
Comment (VoldarGames) [reply VAS.Core/App.cs]: I forgot :) Thanks!

File: VAS.Services/ViewModel/AboutVM.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (vmartos) [reply VAS.Services/ViewModel/AboutVM.cs]: 👍

**Pull request merged**